resources/petajakarta/flood-keywords.xml

```diff
@@ -175,51 +175,45 @@
           <hazard_category>
             <gco:CharacterString>single_event</gco:CharacterString>
           </hazard_category>
-          <raster_hazard_classification>
-            <gco:CharacterString/>
-          </raster_hazard_classification>
           <multipart_polygon>
             <gco:Boolean/>
           </multipart_polygon>
-          <prob_fatality_mag>
-            <gco:Tuple/>
-          </prob_fatality_mag>
-          <volcano_name_field>
+          <thresholds>
+            <gco:Dictionary/>
+          </thresholds>
+          <continuous_hazard_unit>
             <gco:CharacterString/>
-          </volcano_name_field>
-          <field>
-            <gco:CharacterString>state</gco:CharacterString>
-          </field>
-          <value_map>
-            <gco:Dictionary>{&quot;high&quot;: [4], &quot;medium&quot;: [3], &quot;low&quot;: [1, 0, 2]}</gco:Dictionary>
-          </value_map>
-          <vector_hazard_classification>
-            <gco:CharacterString>generic_vector_hazard_classes</gco:CharacterString>
-          </vector_hazard_classification>
+          </continuous_hazard_unit>
+          <inasafe_fields>
+            <gco:Dictionary>{&quot;hazard_value_field&quot;: &quot;state&quot;}</gco:Dictionary>
+          </inasafe_fields>
+          <inasafe_default_values>
+            <gco:Dictionary/>
+          </inasafe_default_values>
+          <scale>
+            <gco:CharacterString/>
+          </scale>
+          <source>
+            <gco:CharacterString>PetaBencana</gco:CharacterString>
+          </source>
           <layer_geometry>
             <gco:CharacterString>polygon</gco:CharacterString>
           </layer_geometry>
+          <value_map>
```

Comment:
ah, I didn't see your comment before commenting. poor connection.
I think we should change it for consistencies.